consensus/ibft/ibft.go

```diff
@@ -495,7 +494,7 @@ func (i *Ibft) writeTransactions(gasLimit uint64, transition transitionInterface
 // If it turns out that the current node is the proposer, it builds a block, and sends preprepare and then prepare messages.
 func (i *Ibft) runAcceptState() { // start new round
 	logger := i.logger.Named("acceptState")
-	logger.Info("Accept state", "sequence", i.state.view.Sequence)
+	logger.Info("Accept state", "sequence", i.state.view.Sequence, "round", i.state.view.Round+1)
```

Comment:
Why the `+1` here?

Reply:
this is because we count rounds from 0, but don't print them out as starting from 0, but on. I saw @Kourin1996 made the same change on the line 748 so I just added it in all places for consistency.